ethereum/dex/trades.sql

```diff
@@ -655,6 +655,27 @@ WITH rows AS (
 
         UNION ALL
 
+        -- 1inch LP
+        SELECT
+            evt_block_time,
+            '1inch LP' AS project,
+            '1' AS version,
+            'DEX' AS category,
+            sender AS trader_a,
```

Comment:
You might want to keep this as NULL so that the outer query coalesces it back to transaction sender. As is this will get you the 1inch router as `trader_a` most of the time. See:
https://github.com/duneanalytics/abstractions/blob/e486226de2019b13aab0734f64167bcc06dc1d19/ethereum/dex/trades.sql#L685

Reply:
1inch Router is not only called of 1inch Liquidity Protocol swap, wdyt?

Reply:
What I mean is that as is this will get you a lower count of unique actual traders if one was to base that count on the unique addresses in the `trader_a` field, because from the events's POV the `sender` is the router, not the EOA that started the trasaction. And you probably want the latter in this table.

Reply:
But this is true num of users when 1inch uses 1LP - it is single user. Else outer statistics should rely on tx_from. I will build separate dashboard for nested volumes.